test/e2e/user_certs_test.go

```diff
@@ -205,7 +210,15 @@ func TestNamedCertificates(t *testing.T) {
 	}
 
 	// wait until a new version has been rolled out with the new configuration
-	testlibraryapi.WaitForAPIServerToStabilizeOnTheSameRevision(t, kubeClient.Pods(operatorclient.TargetNamespace))
+	for {
```

Comment:
how does this exit if the test times out?

Reply:
go test causes a panic in the test executable.